catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java

```diff
@@ -1355,6 +1358,15 @@ public Action createTapAtAction(Sprite sprite, Formula x, Formula y) {
 		TapAtAction action = Actions.action(TapAtAction.class);
 		action.setPosition(x, y);
 		action.setSprite(sprite);
```

Comment:
`action.setSprite(sprite);` should be called before  `action.setPosition(x, y);`